arelle/CntlrWinMain.py

```diff
@@ -816,15 +819,16 @@ def fileOpenFile(self, filename, importToDTS=False, selectTopView=False):
         if filename:
             for xbrlLoadedMethod in pluginClassMethods("CntlrWinMain.Xbrl.Open"):
                 filename = xbrlLoadedMethod(self, filename) # runs in GUI thread, allows mapping filename, mult return filename
-            filesource = None
+            filesource, entrypointFiles, success = parseEntrypointFileInput(self, filename, autoSelect=False)
```

Comment:
Do [the inlineXbrlDocumentSet plugin file menu open dialogs](https://github.com/Arelle/Arelle/blob/b65047cabd0c79cfbd0920e021ec4db0998cebea/arelle/plugin/inlineXbrlDocumentSet.py#L521-L528) still serve a purpose after these changes?

Right now, the standard GUI open dialog only works for non-iXDS files (or EDINET manifests, since both are supported by the same plugin). To open iXDS reports, or any report that relies on a manifest, you have to use the separate file menu entries added by the inlineXbrlDocumentSet plugin (this is obviously undesirable). But that's no longer the case after these changes, you just have to have the inlineXbrlDocumentSet plugin enabled?